src/website/views/components/tabs.pug

```diff
@@ -315,3 +316,94 @@ h4 Large
         <a href="#">Tab Link</a>
       </li>
     </ul>
+
+
+h3 JavaScript
+p.-text
+| Chi library provides a vanilla JavaScript file that enables default tabs functionality.
+| You have to link the tabs.js file and initialize every single tab panel you want to be activated.
+
+h4 Example
+.example.-mb3
+  .-p3.-pb4
+    ul#example-tabs.a-tabs
+      li.-active
+        a(href='#a') Tab a
+      li
+        a(href='#b') Tab b
+      li
+        a(href='#c') Tab c
+    #a.a-tabs-panel.-active
+      p Content for tab a
+    #b.a-tabs-panel
+      p Content for tab b
+    #c.a-tabs-panel
+      p Content for tab c
+  :code(lang='html')
+    <ul class="a-tabs" id="example-tabs">
+      <li class="-active"><a href="#a">Tab a</a></li>
+      <li><a href="#b">Tab b</a></li>
+      <li><a href="#c">Tab c</a></li>
+    </ul>
+
+    <div class="a-tabs-panel -active" id="a">
+      <p>Content for tab a</p>
+    </div>
+    <div class="a-tabs-panel" id="b">
+      <p>Content for tab b</p>
+    </div>
+    <div class="a-tabs-panel" id="c">
+      <p>Content for tab c</p>
+    </div>
+
+    <script src="https://assets.ctl.io/chi/0.8.2/tabs.js"></script>
```

Comment:
Can we make the tabs.js requirement more prominent by removing it from the two examples and placing it into it's own box right after the "Chi library provides a vanilla JavaScript..." paragraph.
```
.-mb2
  :code(lang='html')
    <script src="https://assets.ctl.io/chi/0.8.2/tabs.js"></script>
```